test/servo.js

```diff
@@ -180,6 +180,27 @@ exports["Servo"] = {
     test.done();
   },
 
+  degreeChange: function(test) {
+    test.expect(2);
+
+    this.servo = new Servo({
+      pin: 11,
+      board: board
+    });
+
+    this.servo.to(0);
+
+    this.servo.to(180, 1000, 100);
+    this.servo.to(180, 1000, 100);
+
+    this.clock.tick(1010);
```

Comment:
> The test needs to wait for the first one to complete before calling the second, 

In this case there is no need to wait, since any `servo.to(X)` call that follows, where `X` is the same value, should just: 

1. queue an async `this.emit("move:complete")`
2. return immediately